packages/calcite-components/conventions/Documentation.md

```diff
@@ -15,7 +11,7 @@ Follow these conventions when adding or editing API reference:
 - For plural context, use `calcite-button`s instead of `calcite-button` elements.
 - Use backticks (`` ` ``) for the names of slots, events, properties, CSS variables, and component names (e.g. `calcite-button` instead of calcite-button and `selectionMode` instead of "selectionMode"). Also use backticks for the values of properties and event details (e.g. `true`). If the value is a string, use both backticks and double quotes (e.g. `"single-persist"`).
 - Only use single quotes (`'`) as apostrophes.
-- No links or URLs allowed in descriptions. If a link is necessary, a [custom JSDoc tag](https://stenciljs.com/docs/docs-json#custom-jsdocs-tags) should be added and parsed in the SDK site.
+- No links or URLs allowed in descriptions. If a link is necessary, a custom JSDoc tag should be added and parsed in the SDK site.
```

Comment:
was this a stencil or a docs website restriction?
the new component reference (used by map-components in 4.32, and hopefully Calcite in 4.33) does not have such restriction

Reply:
I believe this was a docs website restriction cc @macandcheese